test/skipTake.js

```diff
@@ -0,0 +1,41 @@
+import { assert } from 'chai';
+import { range } from 'ramda';
+import * as R from 'ramda';
+
+import * as RA from '../src';
+
+describe('skipTake', function() {
+  context('given array as input', function() {
+    specify('should return a valid array', function() {
+      assert.deepEqual(RA.skipTake(3, range(0, 20)), [0, 3, 6, 9, 12, 15, 18]);
+    });
+  });
+
+  context('given empty array as input', function() {
+    specify('should return an empty array', function() {
+      assert.deepEqual(RA.skipTake(3, []), []);
+    });
+  });
+
+  context('given negative count as input', function() {
+    specify('should return an empty array', function() {
+      assert.deepEqual(RA.skipTake(-1, []), []);
+    });
+  });
+
```

Comment:
I'm still missing test for negative skip and non empty array. Not sure how the function behave in that scenario according to the tests.

Reply:
negative skip? I'm not sure i follow that one

Reply:
the first test test for a non empty array?

Reply:
I'm talking about this usecase. I'm not sure what will be the result of this.
```js
RA.skipTake(-2, [1, 2, 3, 4])
```

Reply:
very good point.. and right now it returns [1,3].
What do you believe should be the correct behaviour here? throw an exception or return an empty array?

Reply:
I believe what we see is expected behavior. I'd just create a tests which shows that it doesn't really matter if the number is positive or negative integer.

Another thing that comes to my mind is how does the function reacts on providing `2.3` number instead of `2`?

Reply:
Tests tests tests...they gives all the answers ;]